tests/protocol/test_hostplugin.py

```diff
@@ -33,7 +33,7 @@
 from azurelinuxagent.common.utils import restutil
 from tests.protocol import mock_wire_protocol
 from tests.protocol.mockwiredata import WireProtocolData, DATA_FILE, DATA_FILE_NO_EXT
-from tests.protocol.test_wire import MockResponse
+from tests.protocol.test_wire import MockResponse as TestWireMockResponse
```

Comment:
This one was so much fun to debug! This test uses 2 entities named MockResponse: the one imported here and another at the bottom of the file.

Which one gets used depends on whether patch is used as a decorator or as a function. Ah, the tangled web we weave!

The import alias takes care of this issue.